yarn.lock

```diff
@@ -6281,27 +6281,30 @@ node-pre-gyp@^0.6.29, node-pre-gyp@~0.6.32:
     tar "^2.2.1"
     tar-pack "^3.4.0"
 
-node-sass@3.4.2:
-  version "3.4.2"
-  resolved "https://registry.yarnpkg.com/node-sass/-/node-sass-3.4.2.tgz#ef61069927f1578ae51408ed60298449c4cdd294"
+node-sass@^3.4:
+  version "3.13.1"
```

Comment:
Maybe try `yarn remove node-sass`, then updating it again? It looks like a duplicate for not reason.